draft-ietf-ohai-ohttp.md

```diff
@@ -1065,20 +1088,47 @@ server. The use of idempotent methods might be of some use in managing replay
 risk, though it is important to recognize that different idempotent requests
 can be combined to be not idempotent.
 
-Idempotent actions with a narrow scope based on the value of a protected nonce
-could enable data submission with limited replay exposure. A nonce might be
-added as an explicit part of a request, or, if the oblivious request and target
-resources are co-located, the encapsulated keying material can be used to
-produce a nonce.
+Even without replay prevention, the server-chosen `response_nonce` field
+ensures that responses have unique AEAD keys and nonces even when requests are
+replayed.
+
+
+### Use of Date for Anti-Replay
 
-The server-chosen `response_nonce` field ensures that responses have unique
-AEAD keys and nonces even when requests are replayed.
+Clients SHOULD include a `Date` header field in encapsulated requests.  Though HTTP requests
```

Comment:
Should we call out that this is only when using binary http as the inner protocol?

Reply:
It seems like we're missing the bigger normative requirement that "that the client and target server SHOULD ensure that the protocol that’s encapsulated isn’t going to be vulnerable to replay attacks". That doesn't seem to be stated anywhere, and that's really the main point.

Reply:
Good point.  I've added some text at the top of the anti-replay section (second paragraph).  Let me know if that makes sense.

Reply:
@martinthomson Yeah, that's exactly the kind of text I was thinking of. Thanks for adding!